src/model/net.py

```diff
@@ -64,32 +71,50 @@ def init_lstm(self):
             elif "weight" in name:
                 nn.init.xavier_normal(param)
 
-    def forward_rnn(self, src: List[tensor], hidden: tuple = None):
-        embedded = self.emb(src)
+    def forward_rnn(
+        self, src: List[tensor], hidden: tuple = None, _length: List = None
+    ):
```

Comment:
src 가 어떤 shape 을 갖고 있는지 docstring 으로 언급해주면 보기에 편할 것 같습니다 ㅎㅎ

```suggestion
    def forward_rnn(
        self, src: List[tensor], hidden: tuple = None, _length: List = None
    ):
    """
    Args:
        src: (batch_size, ...)
        hidden: 
        _length: 
    """
```

Reply:
감사합니다 ~ docstring 추가하겠습니당

Reply:
https://marketplace.visualstudio.com/items?itemName=njpwerner.autodocstring

@aisolab @ssaru @inmoonlight 이거 써볼까 하는데 혹시 써보신분?

Reply:
docstrings과 dimension 추가하였습니다.